lib/warden/jwt_auth/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Warden
   module JWTAuth
-    VERSION = '0.7.0'
+    VERSION = '0.7.1'
```

Comment:
Can we revert this change? It's a change not related to what the commit describes.

Reply:
Good point. Fixed!